randovania/games/dread/exporter/patch_data_factory.py

```diff
@@ -44,6 +44,18 @@
     PickupModel(RandovaniaGame.METROID_DREAD, "PROGRESSIVE_SPIN"): ["powerup_doublejump", "powerup_spacejump"],
 }
 
+_SCENARIO_IDS = {
+    "artaria": "s010_cave",
+    "cataris": "s020_magma",
+    "dairon": "s030_baselab",
+    "burenia": "s040_aqua",
+    "ghavoran": "s050_forest",
+    "elun": "s060_quarantine",
+    "ferenia": "s070_basesanc",
+    "hanubia": "s080_shipyard",
+    "itorash": "s090_skybase",
+}
+
```

Comment:
That's just the region name to scenario id. I don't like having data doubled.
Replace it with the suggestion in the other comment